tensorflow/python/keras/utils/vis_utils.py

```diff
@@ -180,6 +188,16 @@ def model_to_dot(model,
       label = '{}: {}'.format(layer_name, class_name)
     else:
       label = class_name
+  
+    # Rebuild the label as a table including the layer's dtype.
+    if show_dtype: 
+      def format_dtype(dtype):
```

Comment:
It seems that this function is not used here, should the label below be:

```
label = '%s|%s' % (label, format_dtype(dtype))
```

Reply:
Definitely, Scott. Thank you!